### PR TITLE
Addition of new hourly metrics and transfer schema change

### PR DESCRIPTION
**Pull request by Jaynaik19**
## Description

Addition of new metric endpoints: 

- `hourly_transfer_metrics`
- `hourly_transfer_volume`

Transfer schema alteration:
Addition of two columns to transfers, Details: https://github.com/connext/nxtp/issues/1689
- transfer_status_update_by_agent
- transfer_status_message_by_agent

### CODE REVIEW

File: packages/agents/cartographer/poller/db/migrations/20220824094332_metrics_transfer_add_fee_revenue.sql [hunk 1]
```diff
@@ -0,0 +1,130 @@
+-- migrate:up
+
+-- 
+-- Schema change
+-- 
+ALTER TABLE public.transfers
+ADD COLUMN IF NOT EXISTS transfer_status_update_by_agent numeric;
```
Comment (sanchaymittal) [inline packages/agents/cartographer/poller/db/migrations/20220824094332_metrics_transfer_add_fee_revenue.sql:7]: both the fields are going to be string

File: packages/agents/cartographer/poller/db/migrations/20220824094332_metrics_transfer_add_fee_revenue.sql [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+ALTER TABLE public.transfers
+ADD COLUMN IF NOT EXISTS transfer_status_message_by_agent numeric;
+
+-- 
+-- New Hourly Metrics
+-- 
+-- 1. Transfer metrics
+CREATE  OR REPLACE VIEW public.hourly_transfer_metrics AS (
```
Comment (sanchaymittal) [inline packages/agents/cartographer/poller/db/migrations/20220824094332_metrics_transfer_add_fee_revenue.sql:16]: if possible, Can you add the API links in the doc and create a new example file for hourly metrics commands?

**Review by sanchaymittal — CHANGES_REQUESTED** [review]

File: packages/agents/cartographer/poller/db/migrations/20220824094332_metrics_transfer_add_fee_revenue.sql (hunk 1 above)
Comment (Jaynaik19) [reply packages/agents/cartographer/poller/db/migrations/20220824094332_metrics_transfer_add_fee_revenue.sql]: made changes.

File: packages/agents/cartographer/poller/db/migrations/20220824094332_metrics_transfer_add_fee_revenue.sql (hunk 2 above)
Comment (Jaynaik19) [reply packages/agents/cartographer/poller/db/migrations/20220824094332_metrics_transfer_add_fee_revenue.sql:16]: added.

**Review by sanchaymittal — APPROVED** [review]

**Pull request merged**